File: scratch-script.py
# pyrefly: ignore [missing-import]
import duckdb
con = duckdb.connect("database/AmaziaERP.db", read_only=True)
res = con.execute("SELECT sql FROM sqlite_master WHERE type='table' AND name='inventory_table'").fetchall()
print("SCHEMA:", res)

File: scratch-script.mjs
import duckdb from 'duckdb';
import path from 'path';

const dbPath = path.resolve(process.cwd(), 'database/AmaziaERP.db');
const db = new duckdb.Database(dbPath, duckdb.OPEN_READONLY);

db.all("SELECT sql FROM sqlite_master WHERE type='table' AND name='inventory_table'", (err, rows) => {
  if (err) console.error(err);
  else console.log(rows);
});
